### DISCUSSION TITLE
AVIF file as background image

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#232] [Q&A] [answered]
Is it possible to set an AVIF image file in the static folder as background image?

The [documentation](https://hbstack.dev/modules/look-and-feel/background-image/) of the module seems to say it isn't.

**Comment by razonyang** [comment, marked as answer]
No, it doesn't, that module will resize the image resource into multiple sizes to match different screens, but AVIF is not supported to do that.
I'd suggest using `webp` for now.

**Reply by LITUATUI** [reply]
Thanks, will do that.

**Reply by LITUATUI** [reply]
It doesn't seem to work even with background.jpg and background.webp in the assets/images folder.

Already enable it on [params.yaml](https://github.com/LITUATUI/pushevs.com/blob/main/config/_default/params.yaml) and imported the module on [modules.yaml](https://github.com/LITUATUI/pushevs.com/blob/main/config/_default/module.yaml).

**Reply by razonyang** [reply]
Seems cache issue, I can see the background image from your site.

**Reply by LITUATUI** [reply]
Strange because I forced the refresh with Shift + F5 keys, I see that it was generated in the resources folder, but don't see it in any browser (Linux computer or Android mobile).

It does appear on [Pingdom](https://tools.pingdom.com/#6396ef4c26000000).

Maybe it's a browser thing. What's your browser?

**Reply by razonyang** [reply]
Chrome desktop and Firefox Android.

**Reply by LITUATUI** [reply]
Found the bug, background image only shows up with light theme and I always use dark mode.

**Reply by LITUATUI** [reply]
BTW, it would be cool to be able to set different background images depending on light or dark mode.

For example, background-light.jpg and background-dark.jpg

Nonetheless, this isn't a serious problem for me, so feel free to ignore it or take your time. The website is already almost as I want it. I'm truly impressed with your theme.

**Reply by razonyang** [reply]
> background image only shows up with light theme and I always use dark mode.

Actually it's also shown on dark mode, but not obvious, it was intended to match the dark mode, but I'm not sure if this is a good design.

![image](https://github.com/hbstack/theme-cards/assets/17720932/dfb6a66e-fa0f-4d09-b81f-6d4559d7c227)

**Reply by LITUATUI** [reply]
You're right, I missed it.

Thanks all good for now :)

**Reply by LITUATUI** [reply]
Just a small thing.

Background is highly pixelated (bad quality), but I'll try different images to see if it gets better.
